chat-plugins/scavengers.js

```diff
@@ -144,13 +197,86 @@ exports.commands = {
 		scavengers.answers = null;
 		scavengers.participants = {};
 		scavengers.finished = [];
+		scavengers.updatinglb = false;
+		scavengers.numblitzes = 0;
 		if (scavengers.result) {
 			Rooms('scavengers').addRaw('<div class="broadcast-blue"><strong>' + scavengers.result + '</strong></div>');
```

Comment:
I think it should be, since it only uses <strong> and <em> tags (I'm not entirely sure what sanitized means, but I think it is from the wikipedia definition)